pkg/management/http.go

```diff
@@ -69,11 +70,14 @@ func managementErrorHandler(logger log.Logger) func(context.Context, error, http
 		switch e := errors.Cause(err).(type) {
 		case kc_client.HTTPError:
 			w.WriteHeader(e.HTTPStatus)
+			w.Write([]byte(internal.ComponentName + "."))
 		case ConvertLocationError:
 			// 201-Created, even if ConvertLocationError occurs, the creation was a success
 			w.WriteHeader(http.StatusCreated)
+			w.Write([]byte(internal.ComponentName + "."))
 		default:
 			defaultHandler(ctx, err, w)
+			
```

Comment:
Did you wrote a unit test for this new line ? :-D

Reply:
Still pondering whether this would improve or not the test coverage :)